src/cargo/util/auth/mod.rs

```diff
@@ -619,7 +620,7 @@ fn auth_token_optional(
     if let Some(cached_token) = cache.get(url) {
         if cached_token
             .expiration
-            .map(|exp| OffsetDateTime::now_utc() + Duration::minutes(1) < exp)
+            .map(|exp| Timestamp::now() + Duration::from_secs(60) < exp)
```

Comment:
Small thing, but you could do `Timestamp::now() + SignedDuration::from_mins(1)` here.

Reply:
Is there some benefit to use `SignedDuration` instead of std's `Duration`?. Latter is just shorter to write and does the same thing :)

Reply:
You get the from_mins constructor. But otherwise, in this case, no.

Reply:
especially sicne you use the above suggestion in your changes to `crates/cargo-test-support/src/registry.rs`